### PR TITLE
Lint Repo & update package.json

### PR DESCRIPTION
**Pull request by chrismaree**
This PR adds a command to the `package.json` to lint the repo. Along with this, the repo is linted.

### CODE REVIEW

**Comment by mrice32** [comment]
> > LGTM! What about adding a husky pre-commit hook too?
> 
> @nicholaspai had done this already and it works great :)
> I basically just added a command to directly lint the whole repo, rather than just linting staged changes.

ooooh, I see, SGTM! Nice change!

File: features/manage-position/Create.tsx [hunk 1]
```diff
@@ -37,14 +37,14 @@ const Create = () => {
     decimals: collDec,
     allowance: collAllowance,
     setMaxAllowance,
-    balance
+    balance,
```
Comment (nicholaspai) [inline features/manage-position/Create.tsx:40]: @chrismaree @adrianmcli I've noticed that there are trailing commas like this. Any idea why this is "good practice" lol?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "chain": "ts-node -O '{\"module\":\"commonjs\"}' chain.ts"
+    "chain": "ts-node -O '{\"module\":\"commonjs\"}' chain.ts",
+    "lint": "prettier './**/*.{ts,tsx,css,html,md}' --write"
```
Comment (nicholaspai) [inline package.json:9]: So you added this because you wanted to be able to run this over the entire repo to capture non-staged files?

Comment (chrismaree) [reply package.json:9]: yup! I had added some additional functionality but my PR ended up being a lot more than the changes I was making due to the lint diffs :)

**Pull request merged**